### PR TITLE
Update RT Lead Handbook

### PR DESCRIPTION
**Pull request by guineveresaenger**
Fixes https://github.com/kubernetes/sig-release/issues/841

Fixes https://github.com/kubernetes/sig-release/issues/605

Includes fix for https://github.com/kubernetes/sig-release/issues/910 (but did not add that in directly)

Sets timeline mentioned in https://github.com/kubernetes/sig-release/issues/913

Specifically adds Zoom and calendar access as checklist items to the onboarting template for RT and RM Leads

Includes general fixes for the handook, streamlining processes.

/sig release
/area release-team
/cc @alejandrox1 
/cc @tpepper
/cc @justaugustus
/cc @jeefy
/cc @mariantalla

### CODE REVIEW

File: release-team/role-handbooks/release-team-lead/README.md [hunk 1]
```diff
@@ -57,7 +57,8 @@ Release Lead is a very time-consuming role, especially towards the end of the re
 
 - Weeks 1-4: 4-8 hours a week
 - Weeks 5-8: 6-12 hours a week
-- Weeks 9-13: 10 to 25 hours a week, including some after-hours and weekend work
+- Weeks 9-13: 10 to 25 hours a week
+- Release Day: Prepare to spend all day on release-related activities.
```
Comment (tpepper) [inline release-team/role-handbooks/release-team-lead/README.md]: Should this be even more explicit, eg: "Given the choice between working after-hours or weekends, and having the release be later than the initial target, delaying the release is preferred."

File: release-team/role-handbooks/release-team-lead/README.md [hunk 2]
```diff
@@ -66,7 +66,17 @@ Among the specific time commitments you have are:
...
+In addition to the absolute time commitment you make, you must also consider the relative burdens on yourself and your team when establishing meetings. This means scheduling meetings and release events compatible with global working hours and coordinating with a global set of team leads and shadows. 
+A few ways to accopmplish this goal are as follows (but please feel free to implement and add your own solution):
```
Comment (tpepper) [inline release-team/role-handbooks/release-team-lead/README.md]: typo: accomplish

File: release-team/role-handbooks/ci-signal/README.md [hunk 3]
```diff
@@ -181,6 +181,14 @@ Questions to ask sig-<name>:
   - Escalation points are expected to be responsive within 24 hours, and to prioritize test failure issues over other issues.
     - If you don't see this happening, get in touch with them on slack or other means to ask for their support.
 
+### SIG-scalability report
+We've had to delay releases due to failing scalability tests in the past, as these take a while to run. It is therefore important to check in with SIG-scalability on the status of their tests weekly. Ask that a member of SIG-scalability should fill the "Weekly Scalability Report" section in the release team meeting notes with an update, once a week. Additionally, a representative of the Release Team (either a CI Signal team member or a Lead team member) should attend the SIG meeting once a month (beginning of the release, code freeze, and near end of release) and check in on the following things:
```
Comment (tpepper) [inline release-team/role-handbooks/ci-signal/README.md]: suggest changing "take a while" to "can take multiple days".

File: release-team/role-handbooks/ci-signal/README.md [hunk 4]
```diff
@@ -181,6 +181,14 @@ Questions to ask sig-<name>:
...
+  - Clearly communicate deadlines (code freeze, planned release date)
+  - Check in on outstanding issues:
+    - anything that may hold a release 
+    - anything that could prevent the upgrade of another dependency (I.e., go)
```
Comment (tpepper) [inline release-team/role-handbooks/ci-signal/README.md]: Style guide discourage "i.e.", and in the same vein of readability the parenthetical might be more clear as "(for example a change to the golang major/minor version)"

File: release-team/role-handbooks/ci-signal/README.md [hunk 5]
```diff
@@ -181,6 +181,14 @@ Questions to ask sig-<name>:
...
+    - whether there are any other tests they are running - they occasionally run scalability jobs on certain PRs, or on certain special configurations (run job with pinned Go version, alpha feature turned on, etc)
```
Comment (tpepper) [inline release-team/role-handbooks/ci-signal/README.md]: Again suggest "golang" for complete clarity.

**Comment by tpepper** [comment]
Generally looks good to me.  A couple small change suggestions, but those could also be follow on changes.

**Comment by guineveresaenger** [comment]
@tpepper - PTAL!

**Comment by tpepper** [comment]
/lgtm

should squash before merge.

**Pull request merged**